Formula/thrulay.rb

```diff
@@ -17,6 +17,9 @@ class Thrulay < Formula
   end
 
   def install
+    # Fix flat namespace usage
+    inreplace "configure", "${wl}-flat_namespace ${wl}-undefined ${wl}suppress", "${wl}-undefined ${wl}dynamic_lookup"
```

Comment:
Do our patches not apply?

Reply:
Unfortunately not.  This `configure` script has quite a different layout with regards to the `flat_namespace` flags:
```
8397:           allow_undefined_flag='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
8401:               allow_undefined_flag='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
11555:           allow_undefined_flag_CXX='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
11559:               allow_undefined_flag_CXX='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
15055:           allow_undefined_flag_F77='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
15059:               allow_undefined_flag_F77='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
17666:           allow_undefined_flag_GCJ='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
17670:               allow_undefined_flag_GCJ='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
```

Reply:
Hm, weird. No luck with regenerating `configure`?

Reply:
Sadly it fails.  I didn't write this up anywhere but for all these flat namespace fixes I tried the following in order:
1) Use our flat namespace patches.
2) Try to regenerate `configure` with `autoreconf`.
3) Patch out the flat namespace references manually.

So for all the PRs which resort to manual patching, that was because I either couldn't get `autoreconf` to run, or it did run but the `configure` failed to run or broke something `make`/`make install`.  I'm happy to push a commit that tries to regenerate `configure` if you'd like to take a look at the failures.  I was able to fix a few cases where some files were missing (`NEWS` etc.) but in the other cases it was really hard to tell what was wrong.